admin/display/dashboard_widget.php

```diff
@@ -23,10 +23,12 @@ function __construct() {
 		 */
 		function add_dashboard_widget() {
 			if ( current_user_can( 'install_plugins' ) && false !== $this->show_widget() ) {
-				wp_add_dashboard_widget( 'semperplugins-rss-feed', __( 'SEO News', 'all-in-one-seo-pack' ), array(
-					$this,
-					'display_rss_dashboard_widget',
-				) );
+				wp_add_dashboard_widget(
```

Comment:
It appears params like this were missed.

```
wp_add_dashboard_widget(
	'semperplugins-rss-feed',
	__( 'SEO News', 'all-in-one-seo-pack' ),
	array(
		$this,
		'display_rss_dashboard_widget',
	)
);
```

Reply:
i'm not sure what you are referring to here

Reply:
...it was just the proper way to WPCS, but nevermind.